Code/GraphMol/Chirality.cpp

```diff
@@ -2411,15 +2412,16 @@ void legacyStereoPerception(ROMol &mol, bool cleanIt,
           bond->getEndAtom()->getChiralTag() == Atom::CHI_UNSPECIFIED) {
         // see if there is an atropisomer bond connected to this bond
 
-        bool hasAtropisomer = false;
```

Comment:
This method of detecting an atropisomer does not always work - it is possible that the wedge/hash bonds have been removed at this point.   It is better to check explicitly for an atropisomer bond below.